### PR TITLE
add ENCRYPTION_SECRET variable

### PR DESCRIPTION
**Pull request by avoylenko**
- "ENCRYPTION_SECRET" env added
- fix typo in variable name

### CODE REVIEW

**Review by davehorton — APPROVED** [review]

File: lib/utils/encrypt-decrypt.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ const crypto = require('crypto');
 const algorithm = process.env.LEGACY_CRYPTO ? 'aes-256-ctr' : 'aes-256-cbc';
 const iv = crypto.randomBytes(16);
 const secretKey = crypto.createHash('sha256')
-  .update(String(process.env.JWT_SECRET))
+  .update(process.env.ENCRYPTION_SECRET || process.env.JWT_SECRET)
```
Comment (davehorton) [inline lib/utils/encrypt-decrypt.js:5]: Please note how this is failing in the github action flow test.  I think you need to restore wrapping it in a String() to make sure it is always a string.  If so, the same applies to the api-server change I think

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/utils/encrypt-decrypt.js (hunk 1 above)
Comment (avoylenko) [reply lib/utils/encrypt-decrypt.js:5]: Github action is failing due to env variables. Either ENCRYPTION_SECRET or JWT_SECRET must be supplied.

Comment (avoylenko) [reply lib/utils/encrypt-decrypt.js:5]: String(undefined) produces 'undefined' as a string.

Comment (davehorton) [reply lib/utils/encrypt-decrypt.js:5]: ok, how do you propose fixing the failing test then?  I am not going to merge until tests pass

Comment (avoylenko) [reply lib/utils/encrypt-decrypt.js:5]: @davehorton should be fine now

**Review by davehorton — APPROVED** [review]

**Pull request merged**